setup.py

```diff
@@ -74,9 +74,45 @@ def run(self):
         install.run(self)
 
 
+class LlvmliteClean(Command):
```

Comment:
Can't you inherit from setuptools' or distutils' own `clean` command class? I don't think we want to reinvent the removal of `egg-info` or `pyc` files (not to mention any other things you may have overlooked here).